tests/um_pdp_test.py

```diff
@@ -229,5 +241,42 @@ def test_pdp_validate_rule_with_conditions_true(self, mock_validate_rule_with_co
 
         self.assertEqual(j, True, "Validate conditional policy rule should be true")
 
+    @mock.patch('handlers.scim_handler.ScimHandler.modifyAuthServerUrl', side_effect=mocked_modify_auth_server)
+    def test_pdp_validate_valid_issuer(self, mock_validate_valid_issuer, raise_for_status=None):
+        with open('../src/standards/request_template_issuer.json') as json_file:
```

Comment:
change this to a folder local to the "test" folder (as stated in previous comments)